config.default.yaml

```diff
@@ -61,6 +61,7 @@ snapshots:
 # definition of the Coordinate Reference Systems
 crs:
   geo_crs: EPSG:4326 # general geographic projection, not used for metric measures. "EPSG:4326" is the standard used by OSM and google maps
+  copernicus_crs: EPSG:4326 # projection for Copernicus data, used by atlite. "EPSG:4326" is the standard used by OSM and google maps
```

Comment:
I'd be curious on the reasons why you suggesting to have this crs to be customizable.

in my view this is not really an option of the user as copernicus has a static CRS and we may not want the user to change it even by mistake.
If they really know what they are doing, changing a global constraint in a script may not be a big deal most likely.

What do you think?